src/Event/ProcessTwigEvent.php

```diff
@@ -0,0 +1,68 @@
+<?php
+
+namespace Drupal\ckeditor5_sections\Event;
+
+use Symfony\Component\EventDispatcher\Event;
+
+/**
+ * Class ProcessTwigEvent
+ *
+ * @package Drupal\ckeditor5_sections\Event
+ */
+class ProcessTwigEvent extends Event {
```

Comment:
@pmelab Just in case we need to hook into it at some point. I added it because it only took 10 minutes to implement but happy to remove if you think it's unnecessary.